src/components/spreadsheet/utils/config-tables.js

```diff
@@ -144,6 +145,17 @@ export const TABLES_DEFINITIONS = {
                 numeric: true,
                 filter: 'agNumberColumnFilter',
                 fractionDigits: 0,
+                editable: true,
+                cellEditor: NumericalField,
+                cellEditorParams: (params) => {
```

Comment:
the nominal voltage is mandatory in the creation form, it should be mandatory here too i think

Reply:
It's fixed now normally